### PR TITLE
Add badges to user profiles

### PR DESCRIPTION
**Pull request by thomasgwatson**

### CODE REVIEW

File: src/routes/GroupSettings/ModeratorsSettingsTab/__snapshots__/ModeratorsSettingsTab.test.js.snap [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 exports[`AddModerator renders correctly when adding with suggestions 1`] = `
 <div
-  data-stylename="styles.add-moderator styles.adding"
+  data-stylename="styles.adding"
```
Comment (thomasgwatson) [inline src/routes/GroupSettings/ModeratorsSettingsTab/__snapshots__/ModeratorsSettingsTab.test.js.snap:5]: Updates to tests based on Tibet's styling edits (So I assume this is ok)

File: src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap [hunk 2]
```diff
@@ -34,7 +34,7 @@ Object {
   },
   "meta": Object {
     "analytics": Object {
-      "eventName": "Registered",
+      "eventName": undefined,
```
Comment (thomasgwatson) [inline src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap:37]: Tibet, did an update to analytics get pushed in your edits?

Comment (tibetsprague) [reply src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap:37]: hmm, i will investigate

File: src/routes/GroupSettings/ModeratorsSettingsTab/__snapshots__/ModeratorsSettingsTab.test.js.snap (hunk 1 above)
Comment (tibetsprague) [reply src/routes/GroupSettings/ModeratorsSettingsTab/__snapshots__/ModeratorsSettingsTab.test.js.snap:5]: oops yes thanks!

**Review by tibetsprague — APPROVED** [review]

File: src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap (hunk 2 above)
Comment (tibetsprague) [reply src/routes/NonAuthLayoutRouter/Signup/__snapshots__/Signup.store.test.js.snap:37]: oh i think you just needed to `yarn` first

**Pull request merged**